control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -1655,6 +1658,69 @@ func (r *reconciler) ensureCloudResourcesDestroyed(ctx context.Context, hcp *hyp
 	return remaining, errors.NewAggregate(errs)
 }
 
+func (r *reconciler) ensureGuestAdmissionWebhooksAreValid(ctx context.Context) error {
+	log := ctrl.LoggerFrom(ctx)
+
+	cpServices := &corev1.ServiceList{}
+	if err := r.cpClient.List(ctx, cpServices, client.InNamespace(r.hcpNamespace)); err != nil {
+		return fmt.Errorf("failed to list control plane services")
```

Comment:
forgot to pass the err here? `return fmt.Errorf("failed to list control plane services: %w", err)